SU-HSDO/suhumsci
# Fix(STN-842): postcard uniform height in collections fix

# READY FOR REVIEW

## Summary
Postcards contained within collections were not getting the uniform height specified by the `--uniform-height` modifier.

## Urgency
Medium

## Steps to Test
1. `npm run test` inside `docroot/themes/humsci/humsci_basic`.
2. Create a flexible page with a Collections component.
3. Populate the Collection with at least 2 Postcards.
4. Configure "Components Field" to have the added classes `hb-raised hb-raised--uniform-height`.
5. Confirm raised cards are the same height.
6. Verify the Postcards within Collections display correctly on both the Traditional and Colorful themes.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
